benchmarks/bench_fringefit.py

```diff
@@ -0,0 +1,213 @@
+import os
+import sys
+import shutil
+import unittest
+import itertools
+import numpy as np
+
+# For information about parameters that are unexpectedly zero, set
+# VERBOSE to true.  Currently there are none, so this is for developers
+# only
+VERBOSE = False
+from casatools import ms, ctsys, table
+from casatasks import fringefit, flagmanager, flagdata
+
+from casatestutils import testhelper as th
+ctsys_resolve = ctsys.resolve
+
+tblocal = table()
+
+# ASV iteration control (https://asv.readthedocs.io/en/stable/benchmarks.html#benchmark-attributes)
+number = 1            # i.e., always run the setup and teardown methods
+repeat = (1, 2, 30.0) # between 1 and 2 iterations per round w/ soft cutoff (start no new repeats) past 1m
+rounds = 3            # amount of instances a "repeat block" is run to collect samples
+min_run_count = 5     # enforce the min_repeat * rounds setting is met
+timeout = 3600        # conservative 1hr hard cap for duration of a single test execution
+
+datapath = ctsys.resolve('unittest/fringefit/')
+
+class Fringefit_tests():
+    prefix = 'n08c1'
+    msfile = prefix + '.ms'
+    uvfile = 'gaincaltest2copy.ms'
+
+    def setUp(self):
+        shutil.copytree(os.path.join(datapath, self.msfile), self.msfile)
+        shutil.copytree(os.path.join(datapath, 'gaincaltest2.ms'), self.uvfile)
+
+    def tearDown(self):
+        shutil.rmtree(self.msfile)
+        shutil.rmtree(self.prefix + '.sbdcal', True)
+        shutil.rmtree(self.prefix + '-zerorates.sbdcal', True)
+        shutil.rmtree(self.prefix + '.mbdcal', True)
+        # shutil.rmtree(self.prefix + '.mbdcal2', True)
+        shutil.rmtree(self.uvfile, True)
+        shutil.rmtree('uvrange_with.cal', True)
+
+    def time_sbd(self):
```

Comment:
Please add a doc string and refer to the name of the original test case in the string such as we have done for bench_flagdata test cases.

Reply:
I've added those changes now. I also removed one test case from the CorrComb class. The test was using corrcomb='none' which will skip the interesting functionality of the parameter, and was only being used as a basis for comparison in the original test.